cypress/plugins/index.ts

```diff
@@ -0,0 +1,5 @@
+import cucumber from 'cypress-cucumber-preprocessor'
+
+// module.exports = (on) => {
```

Comment:
I am wondering if we need this file, as all the code commented out.

Reply:
I thought the same. I left it to have your point of vue, but I guess this file isn't needed anymore :) . I ket the folder in the Cypress directory just in case but will probably delete it when starting to implement real BDD tests